### PR TITLE
Jb splitfix1

### PR DESCRIPTION
**Pull request by jbutcher21**
This updates corrects the split category when the match could not be found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
-# g2audit
\ No newline at end of file
+# g2audit
+
+## Overview
+
+The [G2Audit.py](G2Audit.py) utility compares two entity resolution result sets and computes the precision, recall and F1 scores between them. It can be used to compare different runs to a 
+truth set to determine which one is best or to determine the full affect configuration change had against prior run of the same data.  There are many articles that describe this including ...
```
Comment (antaenc) [inline README.md:6]: Shouldn't this be effect not affect

Comment (antaenc) [inline README.md:6]: Consider either: 

- a configuration change had
- configuration changes had 

"... a prior run..." 

-

Comment (antaenc) [inline README.md:6]: Dont' use the 3 dots ... like this, use a colon : instead and no space between word and colon

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+* https://senzing.zendesk.com/hc/en-us/articles/360045624093-Understanding-the-G2Audit-statistics
+* https://senzing.zendesk.com/hc/en-us/articles/360050643034-Exploratory-Data-Analysis-4-Comparing-ER-results
+* https://senzing.zendesk.com/hc/en-us/articles/360051016033-How-to-create-an-entity-resolution-truth-set
+
+This project is designed to be used along with ...
```
Comment (antaenc) [inline README.md:11]: Same 3 dots

File: README.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+usage: G2Audit.py [-h] [-n NEWERFILE] [-p PRIORFILE] [-o OUTPUTROOT] [-D]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -n NEWERFILE, --newer_csv_file NEWERFILE
+                        the latest entity map file
+  -p PRIORFILE, --prior_csv_file PRIORFILE
+                        the prior entity map file
+  -o OUTPUTROOT, --output_file_root OUTPUTROOT
+                        the ouputfile root name (both a .csv and a .json file
```
Comment (antaenc) [inline README.md:28]: I'd make it clearer this is a dir not a filenmame. 

-o OUTPUTPATH --output_path
                                 the output path (not a filename, both .csv and .json files will be created in this path)

File: README.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+
+## Contents
+
+1. [Prerequisites](#Prerequisites)
+2. [Installation](#Installation)
+3. [Typical use](#Typical-use)
+4. [Output files](#Output-files)
+
+### Prerequisites
+- python 3.6 or higher
```
Comment (antaenc) [inline README.md:41]: Upper case Python

File: README.md [hunk 5]
```diff
@@ -1 +1,92 @@
...
+
+*Plenty of RAM! This process runs very fast as it loads each data set into memory. This is not a problem if your control or truth set is under a million records.  But if you get into the 
+10s or 100s of million records, you may need to run this on a computer with enough RAM to load both sets into memory at the same time.*
```
Comment (antaenc) [inline README.md:44]: you will need to

File: README.md [hunk 6]
```diff
@@ -1 +1,92 @@
...
+
+### Installation
+
+1. Simply place the the following files in a directory of your choice ...  (Ideally along with G2Snapshot.py and G2Explorer.py)
```
Comment (antaenc) [inline README.md:48]: Simply isn't needed. 3 dots again

**Review by antaenc — CHANGES_REQUESTED** [review]
Few little clean up items

**Comment by jbutcher21** [comment]
Changes made Ant, please review and approve.

**Review by antaenc — APPROVED** [review]

**Pull request merged**